library/libraries.js

```diff
@@ -1361,6 +1361,21 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
             return false;
         }
     },
+    'WordPress': {
+        icon: 'wordpress',
+        url: 'https://wordpress.org/',
+        npm: null,
+        test: function (win) {
+            if (!!document.querySelector('meta[name=generator][content^="WordPress"]')) {
+                var versionName = document.querySelector("meta[name='generator']").getAttribute("content");
+                return { version: versionName.replace(/[^0-9\.]+/g,"") };
```

Comment:
over on https://wordpress.org/news/ i spotted `WordPress 5.2-alpha-44796` so I think we'll need something different..  
`.replace(/^\w+\s/,'')` perhaps?

Reply:
That's right. The version can look like any one of:

* `5.1.2`
* `5.1-beta2`
* `5.0-RC1`
* `5.2-alpha-44742-src`

So yes, just removing `'WordPress '` from the beginning will be the way to go.

Reply:
Ooh didn't know more than just numbered releases could be included. Fixed 👍